tests/scripts/thread-cert/config.py

```diff
@@ -60,8 +60,18 @@
 LINK_LOCAL_ALL_ROUTERS_ADDRESS = 'ff02::2'
 
 DOMAIN_PREFIX = 'fd00:7d03:7d03:7d03::/64'
+DOMAIN_PREFIX_REGEX_PATTERN = '^fd00:7d03:7d03:7d03:'
 DOMAIN_PREFIX_ALTER = 'fd00:7d04:7d04:7d04::/64'
 
+PORT_OFFSET = int(os.getenv('PORT_OFFSET', '0'))
+BACKBONE_PREFIX = f'{0x9100 + PORT_OFFSET:04x}::/64'
```

Comment:
The `9100::` is currently reserved (see [Internet Protocol Version 6 Address Space](https://www.iana.org/assignments/ipv6-address-space/ipv6-address-space.xhtml)). Can we define the backbone prefix under the Unique Local Unicast address space (`fc00::/7`)

Reply:
I used 9100::/ because the 1.2 Test Spec requires the `BBP` (Backbone prefix provided by the LinuxRouter) to be 910b::/64.
Do you mean we should change the Test Spec too?

Reply:
If this is what the Thread Test Spec uses, I am fine with it here. However, I think we should raise the question within Thread Group. Do you mind submitting a JIRA on this?

Reply:
I will submit a JIRA on this issue. Resolving this conversation.